lib/utils/data_utils.py

```diff
@@ -75,67 +75,70 @@ def load_mnist_labels(abs_path_i,filename):
 #------------------------------------------------------------------------------#
 
 #------------------------------------------------------------------------------#
-def model_creator(input_var,target_var,rd=None,rev=None,model_dict=None):
-    model_exist_flag=0
-    if model_dict==None:
-        model_dict={}
-        parser=argparse.ArgumentParser()
```

Comment:
I prefer this sort of input parsing because it allows you to provide help options to the user. Any reason why you changed this?